src/libstore/globals.hh

```diff
@@ -613,11 +613,20 @@ public:
           `/dev`, `/dev/shm` and `/dev/pts` (on Linux), and the paths
           configured with the `sandbox-paths` option. This is useful to
           prevent undeclared dependencies on files in directories such as
-          `/usr/bin`. In addition, on Linux, builds run in private PID,
-          mount, network, IPC and UTS namespaces to isolate them from other
-          processes in the system (except that fixed-output derivations do
-          not run in private network namespace to ensure they can access the
-          network).
+          `/usr/bin`.
+
+          In addition, on Linux, builds run in private PID, mount, network,
+          IPC and UTS namespaces to isolate them from other processes in the
+          system (except that fixed-output derivations do not run in private
+          network namespace to ensure they can access the network).
+
+          On macOS, local port binding is disabled by default when the
+          sandbox is enabled. Derivations that have the
+          `__darwinAllowLocalNetworking` attribute set to `true` will have a
+          sandbox exception added to allow it.
+
+          The macOS sandbox has known limitations, and should not be
+          considered a strong security boundary.
```

Comment:
To be honest I think this applies to the Linux one too. (The recent sandbox escape vulnerability worked on Linux but not macOS, even.)

Reply:
I guess it depends on what you consider an escape. Information exfiltration or communication with a cooperating process outside the sandbox, yeah, but I don’t think there’s a known way to *escalate* out of the macOS sandbox to increase privileges?